sql-to-dbsp-compiler/lib/sqllib/src/lib.rs

```diff
@@ -904,6 +905,30 @@ pub fn tan_d(value: F64) -> F64 {
 
 some_polymorphic_function1!(tan, d, F64, F64);
 
+#[inline(always)]
+pub fn sec_d(value: F64) -> F64 {
+    let cos = value.into_inner().cos();
+    if cos.is_zero() {
```

Comment:
Rust doesn't have sec?
Also, if you just do the division with 0 don't you get the same result?
Or maybe the result can be -infinity sometimes?

Reply:
Actually yeah, its Infinity even if we divide by 0. I was expecting a division by 0 error, but turns out it gives `inf` if the number being divided is positive and `-inf` if the number being divided is negative.